lib/arrow/hastus/export_upload.ex

```diff
@@ -90,11 +90,15 @@ defmodule Arrow.Hastus.ExportUpload do
       {:ok, {:error, "Could not parse zip."}}
   end
 
-  @spec upload_to_s3(binary(), String.t()) ::
+  @spec upload_to_s3(binary(), String.t(), String.t() | integer()) ::
           {:ok, :disabled} | {:ok, String.t()} | {:error, term()}
-  def upload_to_s3(file_data, filename) do
+  def upload_to_s3(file_data, filename, disruption_id) do
     if Application.fetch_env!(:arrow, :hastus_export_storage_enabled?) do
-      do_upload(file_data, filename)
+      timestamp = System.system_time(:second)
```

Comment:
nit(non-blocking): I _personally_ find Unix Timestamps unreasonably annoying to try and read when trying to debug, and an annoying extra step to turn them into something usable when trying to investigate issues, so I'd personally favor something like this
```suggestion
      timestamp = DateTime.utc_now(:second) |> DateTime.to_iso8601()
```

But that does make the tests less straightforward 🫠